Magitek/Logic/Scholar/Buff.cs

```diff
@@ -146,16 +146,16 @@ public static async Task<bool> Aetherflow()
             if (Core.Me.HasAetherflow())
                 return false;
 
-            if (Spells.Aetherflow.Cooldown.TotalMilliseconds > 1500)
-            {
-                Logger.Error("Aetherflow on cooldown");
-                return false;
-            }
+            if (Spells.Aetherflow.Cooldown.TotalMilliseconds < 1500)
+                return await Spells.Aetherflow.Cast(Core.Me);
+
+            if (Core.Me.Pet != null && Spells.Dissipation.Cooldown.TotalMilliseconds < 1500 && Casting.LastSpell == Spells.Aetherflow == false)
```

Comment:
This needs a setting. Not everyone is going to want to automatically dissipate their fairy. Dissipation is a VERY controversial spell. The setting should default to false/off.

Dissipation should happen in its own Buff, and not inside of the Aetherflow. Just set the priority on dissipation higher in the primary routine.

Core.Me.Pet can also be Seraph, in which case, dissipation is disabled, so it needs to check that as well.

Reply:
Might break this apart into separate PRs.